### PR TITLE
Fix #1755

### LINKED ISSUE #1755: Add "official" documentation for using multiple file uploads with SQLite / MySQL
Serialized column + string column...

### PR DESCRIPTION
**Pull request by bananaappletw**
Documentation for SQLite using multiple file uploads.

### CODE REVIEW

**Comment by bananaappletw** [comment]
@amatsuda Could you review this pull request?
Just add the document about different database, nothing with codebase.
Thanks.

**Comment by bananaappletw** [comment]
@mshibuya Could you review it?
Just add the document about different database, nothing with codebase.
Thanks.

File: README.md [hunk 1]
```diff
@@ -144,14 +144,25 @@ column for example. Your choice depends on what your database supports. For
 example, create a migration like this:
 
 
+#### For PostgreSQL
```
Comment (mshibuya) [inline README.md]: This needs to be read generally, like `For databases with ActiveRecord json data type support(e.g. PostgreSQL, MySQL)`

File: README.md [hunk 2]
```diff
@@ -144,14 +144,25 @@ column for example. Your choice depends on what your database supports. For
...
+
 	rails g migration add_avatars_to_users avatars:json
 	rake db:migrate
 
+#### For SQLite
```
Comment (mshibuya) [inline README.md]: And vice versa.

**Comment by bananaappletw** [comment]
@mshibuya Updated, thanks for the review.

**Comment by mshibuya** [comment]
Thanks!

**Pull request merged**